src/SongManager.cpp

```diff
@@ -482,7 +483,7 @@ void SongManager::UnlistSong(Song *song)
 	vector<Song*>* songVectors[3] = { &m_pSongs, &m_pPopularSongs, &m_pShuffledSongs };
 	for (int songVecIdx=0; songVecIdx<3; ++songVecIdx) {
 		vector<Song*>& v = *songVectors[songVecIdx];
-		for (auto i = 0; i < v.size(); ++i) {
+		for (size_t i = 0; i < v.size(); ++i) {

```

Comment:
I suspect we've had this discussion before, but I'd like a reminder. Why was `size_t` explicitly chosen?